### PR TITLE
TDR-2137 Add Closure metadata selected overview page for single file

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

File: test/services/ConsignmentServiceSpec.scala [hunk 1]
```diff
@@ -221,6 +224,67 @@ class ConsignmentServiceSpec extends AnyWordSpec with MockitoSugar with BeforeAn
     getConsignmentDetails should be(GetConsignment(0, None))
   }
 
+  "getConsignmentFileMetadata" should {
+    "return consignment with file metadata when consignment ID and selected file ids are passed" in {
```
Comment (TomJKing) [inline test/services/ConsignmentServiceSpec.scala:228]: Lower case "id" for consistency.

File: test/services/ConsignmentServiceSpec.scala [hunk 2]
```diff
@@ -221,6 +224,67 @@ class ConsignmentServiceSpec extends AnyWordSpec with MockitoSugar with BeforeAn
...
+        Some(gcfm.Variables(consignmentId, Some(FileFilters(None, selectedFileIds, None))))))
+        .thenReturn(Future.successful(response))
+
+      val getConsignmentDetails = consignmentService.getConsignmentFileMetadata(consignmentId, bearerAccessToken, selectedFileIds).futureValue
+
+      getConsignmentDetails.files.size should be(1)
+      getConsignmentDetails.files.head.metadata.foiExemptionCode should be(exemptionCode)
+    }
+
+    "return consignment with file metadata when only consignment ID is passed" in {
```
Comment (TomJKing) [inline test/services/ConsignmentServiceSpec.scala:247]: Lower case "id" for consistency.

**Review by TomJKing — COMMENTED** [review]
Looks good. Just some minor comments.

There are some new designs: https://tdr-prototype.herokuapp.com/metadata/closure-metadata/T2-flour/closure-added, which have been introduced since you started this, so there are some updates to the wording on the page.

File: app/views/standard/additionalMetadataSummary.scala.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
+@import views.html.partials.{transferReference}
+@import graphql.codegen.GetConsignmentFilesMetadata.getConsignmentFilesMetadata.GetConsignment.Files.Metadata
+
+@(fileName: String, consignmentRef: String, metadata: Metadata, name: String)(implicit messages: Messages, request: RequestHeader)
+@main(title = "Edit metadata for a single file", name = name) {
```
[the remark this answers is not in the document]
Comment (vimleshtna) [reply app/views/standard/additionalMetadataSummary.scala.html]: Updated it and added buttons too as per the design, thanks.

File: app/views/standard/additionalMetadataSummary.scala.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  @defining(play.core.PlayVersion.current) { version =>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+          Add or edit closure metadata for a single file
```
[the remark this answers is not in the document]
Comment (vimleshtna) [reply app/views/standard/additionalMetadataSummary.scala.html]: Updated it, thanks.

File: test/services/ConsignmentServiceSpec.scala (hunk 1 above)
Comment (vimleshtna) [reply test/services/ConsignmentServiceSpec.scala]: Right, updated it everywhere in the file, thanks.

**Review by TomJKing — DISMISSED** [review]
Looks good. Thanks

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**